tcf_website/views/browse.py

```diff
@@ -123,7 +126,8 @@ def course_view(request, mnemonic, course_number):
                       'course': course,
                       'instructors': instructors,
                       'latest_semester': latest_semester,
-                      'breadcrumbs': breadcrumbs
+                      'breadcrumbs': breadcrumbs,
+                      'this_semester_instructors': this_semester_instructors
```

Comment:
Seems like all you're using this variable for in the template is to check whether or not it exists — might be better to just have a Boolean like `taught_this_semester`. This is mainly just for readability purposes, as you'd just end up moving the nullchecking from the template to the view.